entries/1/1password.com.json

```diff
@@ -4,12 +4,17 @@
     "tfa": [
       "totp",
       "u2f",
-      "custom-hardware"
+      "custom-hardware",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Duo"
     ],
     "documentation": "https://support.1password.com/two-factor-authentication/",
-    "notes": "Teams using Duo with 1Password can't enable additional 2FA as it's already handled by Duo. Hardware tokens (not U2F) are only available through Duo.",
+    "recovery": "https://support.1password.com/two-factor-authentication/#if-you-lose-access-to-your-authenticator-app",
```

Comment:
It's the same page as `documentation`, just with an anchor - this provides no additional info.
```suggestion
```